.travis.yml

```diff
@@ -1,35 +1,28 @@
 sudo: false
 language: ruby
 cache: bundler
-before_install:
-   bundle -v
+before_install: bundle -v
 script: env COVERAGE=yes bundle exec rake
 matrix:
   fast_finish: true
   include:
-  - rvm: '2.5'
-    env: PUPPET_GEM_VERSION='~> 6.0'
-  - rvm: '2.4'
-    env: PUPPET_GEM_VERSION='~> 6.0'
-  - rvm: '2.3'
-    env: PUPPET_GEM_VERSION='~> 6.0'
-  - rvm: '2.4'
-    env: PUPPET_GEM_VERSION='~> 5.0'
-  - rvm: '2.3'
-    env: PUPPET_GEM_VERSION='~> 5.0'
-  - rvm: '2.1'
-    env: PUPPET_GEM_VERSION='~> 5.0'
-  - rvm: '2.4'
-    env: PUPPET_GEM_VERSION='~> 4.0'
-  - rvm: '2.3'
-    env: PUPPET_GEM_VERSION='~> 4.0'
-  - rvm: '2.1'
-    env: PUPPET_GEM_VERSION='~> 4.0'
-  - rvm: '2.1'
-    env: PUPPET_GEM_VERSION='~> 3.0'
-  - rvm: '2.0'
-    env: PUPPET_GEM_VERSION='~> 3.0'
-  - rvm: '1.9'
-    env: PUPPET_GEM_VERSION='~> 3.0'
+    - rvm: "2.5"
```

Comment:
does it make sense to add ruby2.6 as well? (maybe in another PR)

Reply:
Yes that would be a good idea but I only removed the deprecated items in this request.